tests/unit/lfp/lfp.cpp

```diff
@@ -38,7 +38,7 @@ BOOST_AUTO_TEST_CASE(LFP_PointSource_LineSource) {
     std::array<double, 3> segment_end = segment_start +
                                         std::array<double, 3>{0.0, 0.0, segment_length};
     double floor{1.0e-6};
-    double pi{3.141592653589};
+    pi = 3.141592653589;
```

Comment:
why don't we get pi from neuron here?

Reply:
`CoreNEURON` reads its value from either `NEURON` in on line mode or `CoreNEURON` dataset which is generated by `NEURON` (`globals.dat` file). Since in the unit tests we don't run `NEURON` before, we can't get this value from `NEURON`. `pi` is a global variable in the `coreneuron` namespace, so I thought it would be enough to set it manually here.